ui/bookmarks/reducers.py

```diff
@@ -35,10 +35,12 @@ def go_to_bookmark(self, state, n):
         book_n = state['user']['current_book']
         book = state['user']['books'][book_n]
         bookmarks = book.bookmarks[page * height:(page * height) + height]
+        set_book_page = BookReducers().set_book_page
+        if n == len(bookmarks):
+            return set_book_page(state, book.max_pages)
```

Comment:
It makes it easier to review your changes if you base your branch on `origin/master` rather than including changes from other, unmerged, pull-requests.

Reply:
This really needs to be removed now as that pull-request has been rejected.